### DISCUSSION TITLE
Supply issues

### DISCUSSION THREAD

**Discussion by captstent** [AxisCommunications/discussions#54] [General] [answered]
I know there is a supply issue with getting cameras to the United States. Is there any idea or guess when they might be available? We have used Axis exclusively on our campus but now my director is asking about other brands. If it wasn't too far off, I think I could put him off or avoid the subject.
Thank you.

**Comment by sebaxis** [comment, marked as answer]
Hi Captstent. This is a forum for development & integration with Axis devices. The availability of products can be answered by your sales representative or reseller.

**Reply by drewpacino** [reply]
Hi Captstent, if you need help finding the local Axis sales team please contact me. My email is drew.pacino@axis.com
